generators/firechip/src/main/scala/TargetConfigs.scala

```diff
@@ -378,3 +378,32 @@ class FireSimLargeBoomSV39CospikeConfig extends Config(
   new WithFireSimConfigTweaks++
   new chipyard.config.WithSV39 ++
   new chipyard.LargeBoomV3Config)
+
+class WithFireSimNoTraceIODesignTweaks extends Config(
```

Comment:
Yeah your right

Reply:
This isn't exactly what I meant.... I meant you can just prepend a `chipyard.config.WithNoTraceIO` to any Firechip config that doesn't want TraceIO, instead of creating a `WithNoTraceIOFireSimBridges` and `WithFireSimNoTraceIODesignTweaks`